Erreur de nom : vous semblez utiliser un identifiant qui n'existe pas. Auriez-vous fait une typo ?
NameError("name 'faceAuMur' is not defined",)
  File "<script>", line 50, in <module>
